tests/admin.test.ts

```diff
@@ -182,3 +183,47 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('event creation', () => {
```

Comment:
Can you move this test case into `event.test.ts`? I think we should keep all event-related tests together, despite this being an admin action -- the other event creation test is in that file too. I'd also add another test case with two events where the criteria isn't met and it fails